docs/user/files.rst

```diff
@@ -7,23 +7,38 @@ This works even if it has been changed in the meantime.
 
 .. note::
 
-    The available options might be limited by :ref:`privileges`.
+    What options are available might be limited by :ref:`privileges`.
+
+.. _download:
 
 Downloading translations
 ------------------------
 
 From the project or component dashboard, translatable files can be downloaded
-using the :guilabel:`Download original translation file` in the :guilabel:`Files`
-menu, producing a copy of the original file as it is stored in the upstream
-Version Control System.
-
-You can also download the translation converted into one of widely used
-localization formats. The converted files will be enriched with data
-provided in Weblate such as additional context, comments or flags.
+in the :guilabel:`Files` menu.
 
-Several file formats are available, including a compiled file
-to use in your choice of application (for example ``.mo`` files for GNU Gettext) using
-the :guilabel:`Files` menu.
+The first option is to download the original format as translated in the
+repository. In this case, any pending changes in the translation are committed
+and the up-to-date file is downloaded without conversion.
```

Comment:
Some improvement:

```suggestion
The first option is to download the file in the original format as it is stored in the
repository. In this case, any pending changes in the translation are getting committed
and the up-to-date file is yield without any conversions.
```

Those are same as before, I believe all of them makes the snippet better, but I don't insist on any of those...